contributing_to_docs/contributing.adoc

```diff
@@ -101,15 +101,15 @@ This first line is unconditionalized, and will appear for all versions.
 
 \ifdef::openshift-online[]
 This line will only appear for OpenShift Online.
-endif::[]
+\endif::[]
 
 \ifdef::openshift-enterprise
```

Comment:
So the endifs are now showing up, but there are still slashes rendering before the second and third ifdefs. Suggest trying to play with this a little more to see if you can get rid of that as well.